Khan/khan-dotfiles
# INFRA-8460 Initial kabuild container

* All dependencies necessary to run "make deps" in webapp
  * "make run" maps ssh-key & ~/khan appropriately to access git, get webapp, etc.
* Try to do everything that linux-setup.sh does
* Enough debug tools that this container is useful for debugging things
    
Issue: https://khanacademy.atlassian.net/browse/INFRA-8460
    
Test plan:
    
Run the following commands:
- make build
- make run_dirty
    
Check that ~/khan contains local ~/khan and that git works.
This is likely very slow on a mac, so be careful.
    
Ideally, try a clean checkout of webapp somewhere followed by
"make deps". This is probably best tested on a linux machine.
    
Also test "make run"
